main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1458,13 +1458,19 @@ private void updateWatchlistBox() {
          */
         private void updateFavPointBox() {
             // Favorite counts
-            if (cache.getFavoritePoints() > 0) {
-                favoriteLine.left.setVisibility(View.VISIBLE);
-                favoriteLine.right.setText(res.getString(R.string.favorite_count, cache.getFavoritePoints()));
-            } else {
+            final int favCount = cache.getFavoritePoints();
+            if (favCount < 0) {
                 favoriteLine.left.setVisibility(View.GONE);
-            }
+            } else {
+                favoriteLine.left.setVisibility(View.VISIBLE);
+                // Special case when cache has no fav points
+                favoriteLine.right.setText(res.getString(R.string.favorite_count, favCount));
 
+                final int findsCount = cache.getFindsCount();
+                if (favCount > 0 && findsCount > 0) {
+                    favoriteLine.right.setText(res.getString(R.string.favorite_count_percent, favCount, (float) (favCount * 100) / findsCount));
```

Comment:
The format string expects an integer as last argument.

Reply:
updated format string in case someone will want to show digits after dot :)